### ISSUE TITLE
Download function not working

### ISSUE THREAD

**Issue by niklas-8** [shaqian/flutter_ssh#14]
When using the `sftpDownload` function the app throws this exception:
```
E/SshPlugin(25046): Failed to download /home/niklas/.bashrc
E/flutter (25046): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: PlatformException(download_failure, , null)
```
Tested on Xiaomi Mi 9T, Android 9, flutter 1.7.8, ssh 0.0.4

**Comment by niklas-8** [comment]
It's working with [this patch](https://github.com/sjagat/flutter_ssh/commit/9e109577c6a7b8272d2159b4b86b6fe46f5c194f) that was made to fix [this issue](https://github.com/shaqian/flutter_ssh/issues/15).

**Comment by shaqian** [comment]
This is fixed 0.0.5. Archiving the issue.

Thanks,
Qian
